src/utils/schema.ts

```diff
@@ -28,3 +32,47 @@ export function parseSchemaResult(schema: z.ZodTypeAny, v: unknown) {
   }
   return schemaResult;
 }
+
+export async function validateAnt({
+  processId,
+  ao = connect(),
+}: {
+  processId: string;
+  ao: AoClient;
+}): Promise<Record<AoANTHandler, { valid: boolean; error?: string }>> {
```

Comment:
would it be helpful to be able to provide an ANT instance here?

Reply:
Actually yea thats a good idea.